tests/e2e/cypress/support/index.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  initDataResources,
+  setUserTokenApiV1,
+  setUserTokenApiV2,
+  submitResultApiClapi,
+  removeDataResources,
+  applyCfgApi,
+} from './centreonData';
+
+before(() => {
+  setUserTokenApiV1();
+  setUserTokenApiV2();
+
+  initDataResources().then(() => {
+    applyCfgApi().then(() => {
+      // Necessary to wait checks on the engine
+      // eslint-disable-next-line cypress/no-unnecessary-waiting
+      cy.wait(5000);
```

Comment:
you may need to find another way to check engine is available, cause it's a bit long to wait 5s each time configuration is applied

Reply:
instant is not possible, so do you have an idea or a suggestion on this subject?

Reply:
Is there a way to listen to a change within a file using a command within the docker?

Reply:
@kduret was proposing to do an assert too, can you help me find the right information about it?

Reply:
2 ways : 
- wait until services appear in `centreon_storage.services` table (faster)
- go in `resources status` page and wait until services appear in listing

Reply:
jumping in, I see two things here
checking that engine is available (eg centreon system service is running and has effectively restarted)
checking that configuration has been applied (checking db as kdu suggested)

are both handled?

Reply:
> 2 ways :
> 
> * wait until services appear in `centreon_storage.services` table (faster)
> * go in `resources status` page and wait until services appear in listing

Ok. so, What would allow to validate that the sending of "submit results" of a status on a service is well taken into account?
Status change action on : 
[URL_CENTREON_API]?action=submit&object=centreon_submit_results